internal/server/v2/observation/contained_in.go

```diff
@@ -88,13 +87,36 @@ func FetchContainedIn(
 	if queryDate != "" {
 		readCollectionCache = hasCollectionCache(ancestor, childType)
 		if readCollectionCache {
-			btData, err := stat.ReadStatCollection(
+			var btData map[string]*pb.ObsCollection
+			var err error
+			btData, err = stat.ReadStatCollection(
 				ctx, store.BtGroup, bigtable.BtObsCollection,
 				ancestor, childType, variables, queryDate,
 			)
 			if err != nil {
 				return nil, err
 			}
+			// When child places have multiple types, Bigtable only stores the data
+			// for one of the types. This is to use a fallback type when there is no
+			// data for the queried type.
+			// TODO: remove this logic once all types data are computed.
+			if childType == "AdministrativeArea1" {
+				refetch := true
+				for _, data := range btData {
+					if data != nil {
+						refetch = false
+					}
+				}
+				if refetch {
+					btData, err = stat.ReadStatCollection(
+						ctx, store.BtGroup, bigtable.BtObsCollection,
+						ancestor, "State", variables, queryDate,
+					)
+					if err != nil {
+						return nil, err
+					}
+				}
+			}
```

Comment:
nit: Wonder if we can do like:
{
  "AA1": "State",
  "State": "AA1"
}

If childType in the map, use the value?

Reply:
As this is only for temp fix before Flume fix, will not make this a general solution for now. Also looks like we only need to check when it's AA1, since State is preferred in Flume from table_types.textproto?